ui/manual.py

```diff
@@ -126,6 +126,12 @@ def draw(self, context):
             row = col.row(align=True)
             row.scale_y = button_height
             row.operator(Armature_manual.OptimizeStaticShapekeys.bl_idname, icon='MESH_DATA')
+            
+            if not Common.version_2_79_or_older():
```

Comment:
We should probably add a 'how to use' button like we did for Bake (just link to the FA page I guess) that explains how to use it, but we can deal with all that after the merge